miniscope_io/stream_daq.py

```diff
@@ -563,6 +641,15 @@ def capture(
                 if writer:
                     picture = cv2.cvtColor(image, cv2.COLOR_GRAY2BGR)  # If your image is grayscale
                     writer.write(picture)
+                if show_metadata:
+                    current_time = time.time()
+                    if current_time > next_time:
+                        try:
+                            header_plotter.update_plot(metadata_list)
```

Comment:
This is probably thread/process-unsafe, but is not causing problems at the moment. It's fine to miss a couple of data here so just doing error handling as we go might be fine if Locking these every time is too much.